ccswbs/gus
# Add Background Color option to Media and Text widget

# Summary of changes
Add Background Color option to Media and Text widget so they can be used instead of custom-built YAML components

## Frontend
- add relevant background color fields to `gatsby-node.js` schema
- add background and text color logic to `mediaText.js`
- add `data-title` fields to `widgets.js` and `sectionWidgets.js` for reference purposes (unlike the html `title` attribute, `data-title` does not result in a tooltip on hover and is invisible to end users unless they view source code)
- remove obsolete media links field
- upgrade packages

## Backend
- add Background Color taxonomy with choices based on Bootstrap 5 classes
- add `Background` field to Media and Text paragraph 
- remove obsolete links field from Media and Text paragraph

# Test Plan

- Go to https://tqtest.gatsbyjs.io/media-text-testing and ensure it's consistent with the original version on https://preview-ugconthub.gtsb.io/media-text-testing
- Go to https://tqtest.gatsbyjs.io/media-text-test2 and ensure the colors display correctly (headings and text should be black for uog-blue-muted and light-gray backgrounds, white for dark-gray backgrounds, and default otherwise, i.e. dark text, red heading)
- Do the same for https://tqtest.gatsbyjs.io/media-text-video
- Review https://tqtest.gatsbyjs.io/bcomm/become-a-global-leader and ensure the two `Future You` media/text widgets have the uog-blue-muted background color
- Review https://tqtest.gatsbyjs.io/study-in-canada and verify the `Things You Should Know About the City of Guelph` media and text widgets look and behave like the custom YAML widgets on https://www.uoguelph.ca/study-in-canada
- Drupal multidev can be reviewed at https://medtxtbg-chug.pantheonsite.io/
